Description: sendmail path
 The current code uses /usr/lib/sendmail and then /usr/sbin/sendmail as
 a fallback. Policy says it should use /usr/sbin/sendmail.
Origin: vendor
Forwarded: not-needed
Author: Santiago Vila <sanvila@unex.es>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2019-03-24
Bug-Debian: https://bugs.debian.org/539193

--- a/lib/Mail/Mailer.pm
+++ b/lib/Mail/Mailer.pm
@@ -26,7 +26,7 @@
 sub Version { our $VERSION }
 
 our @Mailers =
-  ( sendmail => '/usr/lib/sendmail;/usr/sbin/sendmail;/usr/ucblib/sendmail'
+  ( sendmail => '/usr/sbin/sendmail;/usr/lib/sendmail;/usr/ucblib/sendmail'
   , smtp     => undef
   , smtps    => undef
   , qmail    => '/usr/sbin/qmail-inject;/var/qmail/bin/qmail-inject'
